lib/nodejs/kml-handler.js

```diff
@@ -0,0 +1,43 @@
+// kml-handler.js
```

Comment:
I think that we need to put our ITDG header at the top of this file since it was newly created by this program.